base_layer/core/src/transactions/transaction.rs

```diff
@@ -200,7 +201,7 @@ pub enum TransactionError {
 
 /// An unblinded output is one where the value and spending key (blinding factor) are known. This can be used to
 /// build both inputs and outputs (every input comes from an output)
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
This is potentially a problem as it can leak a key, when we are serializing outputs?

Reply:
This was done because `pub unblinded_change_output: Option<UnblindedOutput>,`  was added to `struct RawTransactionInfo` to minimize code duplication in other places.  
`struct RawTransactionInfo` already contains a lot of secret information and is dependent on the `Serialize, Deserialize` traits.